app/assets/stylesheets/components/breadcrumbs.scss

```diff
@@ -65,4 +80,61 @@
   .bullet {
     margin: 0 6px;
   }
+}
+
+
+// .breadcrumbs {
+//     ul:first-of-type {
+//       width: 500px;
+//       white-space: nowrap;
+//     }
+
+//     ul li a {
+//       white-space: nowrap;
+//     }
+// }
+
```

Comment:
Can we remove this commented out code?

Reply:
Should be removed in https://github.com/chapmanu/cascade-assets/pull/658/commits/c82a9100661fa744cdff1867410f38edadb126e1